app/models/user.rb

```diff
@@ -41,5 +40,21 @@ def set_default_role
   has_one :profile
   has_many :application_letters
   has_many :requests
-       
+
+  # Returns the number of previously accepted applications from the user
+  #
+  # @param none
+  # @return [Int] of number of accepted applications
```

Comment:
What about temporary accepted applications?
Is it intended those are included here? (@tobiduer)

If not, I suggest to update the code (even if it means you need to change the database to figure out whether an accept/reject is final), or at least add a TODO to indicate this function needs to be updated later.